sdcm/gemini_thread.py

```diff
@@ -148,6 +183,13 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
             results_copied = docker.receive_files(src=self.gemini_result_file, dst=local_gemini_result_file)
             assert results_copied, "gemini results aren't available, did gemini even run ?"
 
+            local_gemini_test_statements_file = os.path.join(
```

Comment:
@CodeLieutenant 

I've added this into your PR, copying those files back

Reply:
I'm not sure why we need this copy, as these files are mounted in docker container, we should have them inside $HOME

Reply:
it's on the loader machine $HOME, not where the SCT test is running.

we are collecting the logs from where the test is running in this case, we can optimize it later.
but now for simplicity it's the fastest way to collect it as as the gemini_results fail is being collected.